codegen/smithy-dafny-codegen-test/src/test/java/software/amazon/polymorph/smithyrust/RustTestModels.java

```diff
@@ -30,8 +31,19 @@ class RustTestModels extends TestModelTest {
     DISABLED_TESTS.add("aws-sdks/lakeformation");
     DISABLED_TESTS.add("aws-sdks/sqs");
     DISABLED_TESTS.add("aws-sdks/sqs-via-cli");
+    //TODO: Add support for Recursive shapes.
+    DISABLED_TESTS.add("RecursiveShape");
+    // V2 Models are not yet supported in Rust.
+    DISABLED_TESTS.add("aws-sdks/ddbv2");
+    DISABLED_TESTS.add("aws-sdks/kmsv2");
```

Comment:
Don't these ones work for all current languages now? Bad merge perhaps?

Reply:
The setup is not there.